content/posts/newsletter-020/index.md

```diff
@@ -64,6 +64,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [pGLOWrpg]
+
+![Improved text rendering](pglowrpg_progress.jpg)
+
+[pGLOWrpg] by [@Roal_Yr] is a Procedurally Generated Living Open World RPG,
+a long-term project in development, which aims to be a narrative text-based game
+with maximum portability and accessibility.
+
+Recent updates include:
+- Reformatting a printing interface, which makes it very easy to link text UI and code.
+- Switching to .ron file format for storing configs and strings.
+- Implementing individual strings coloring for better visual perception.
+
+_Discussions: [Twiter](link)_
```

Comment:
This link is broken - it would need to either be `[Twitter]` or `[Twitter](Twitter)` to connect with the URL at the bottom of the section.

Also, I would recommend changing this to `[Twitter](pGLOWrpg-twitter)` and then changing the URL bit to `[pGLOWrpg-twitter]: https://twitter.com/pglowrpg`, to avoid collisions with other posts' links. IIRC, link IDs are global to the entire post, so a link that's just marked as `twitter` could get mixed up.

Reply:
Fixing.